orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/DetermineRollbackCandidatesTask.java

```diff
@@ -105,172 +105,120 @@ public long getTimeout() {
   @Nonnull
   @Override
   public TaskResult execute(@Nonnull StageExecution stage) {
-    Map<String, String> rollbackTypes = new HashMap<>();
-    Map<String, Map> rollbackContexts = new HashMap<>();
-
     StageData stageData = stage.mapTo(StageData.class);
-    Map<String, Object> cluster;
-
     AtomicReference<Moniker> moniker = new AtomicReference<>(stageData.moniker);
```

Comment:
I'm not sure why this is an `AtomicReference`.  Could multiple threads be accessing this code simultaneously?

Reply:
@ajordens Do you have any idea why this was originally an atomic reference?

Reply:
yeah, this doesn't make sense to me. all this code is single threaded

Reply:
Fixed in b6e3f91